gulpfile.js

```diff
@@ -113,6 +114,24 @@ gulp.task('typings', (done) => {
   done();
 });
 
+gulp.task('themes', (done) => {
+  gulp
+    .src('tokens/themes/back-office-web.yml')
+    .pipe($.rename(addPrefix))
+    .pipe(
+      $.theo({
+        transform: {type: 'web'},
+        format: {type: 'rails.yml'},
```

Comment:
Let’s try making a custom formatter that transforms the variants into token format and then use the existing formatters for outputting different data types